### DISCUSSION TITLE
Sandboxing

### DISCUSSION THREAD

**Discussion by avidseeker** [neomutt/neomutt#4338] [Development]
Are there any neomutt users using isolation/sandboxing software like [bubblewrap](https://wiki.archlinux.org/title/Bubblewrap)? Would you mind sharing your set-up?

**Comment by avidseeker** [comment]
A suitable topic to add to https://neomutt.org/guide/security#algolia:div%20%3E%20p:nth-of-type(1)

**Comment by fira959** [comment]
If it does not need to be bubblewrap, there is an alternative sandboxing project using apparmor
https://github.com/roddhjav/apparmor.d

It also includes a profile for mutt

The apparmor.d repo should probably be added to the sandboxing page as well.

**Comment by avidseeker** [comment]
I found one:

https://github.com/StandingPadAnimations/sandboxing-scripts/blob/main/neomutt
